src/main/aem/jcr_root/apps/cq/core/content/nav/.content.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="nt:unstructured">
+    <tools jcr:primaryType="nt:unstructured">
+        <security
```

Comment:
Please remove `jcr:title` and `id` property, so that everything will be inherited through Sling Resource Merger